app/views/billing_expenses/show.html.slim

```diff
@@ -79,3 +79,61 @@
             = t_attr(:created_by)
           .col-xs-6
             = @billing_expense.user.profile.contact.full_name
+
+.row
+  .col-xs-7.col-xs-offset-1
```

Comment:
Why use bootstrap grid system in order to get a margin on something that is intended to be on pdf?
And `.table-responsive` has also questionable sense here.